spec/lib/evss/letters/service_spec.rb

```diff
@@ -21,5 +21,14 @@
         end
       end
     end
+
+    describe '#download_letter' do
+      it 'downloads a pdf' do
+        VCR.use_cassette('evss/letters/download') do
+          response = subject.download_letter_by_type(EVSS::Letters::Letter::LETTER_TYPES.first)
+          expect(response).to include('%PDF-1.4')
```

Comment:
👍 I'll be it in a let, it's checking that the body of the response is a PDF file (string)